indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -195,4 +195,27 @@
             $(this).closest('tr').toggleClass('selected', this.checked);
         }).trigger('change');
     };
+
+    global.showThemeSettingsForm = function showThemeSettingsForm(options) {
+        $('#timetable_theme').on('change', function() {
```

Comment:
take this ID from `options`.

also, check what we do in other places to specify the options that are available (`$.extend(..)` with some more or less useful defaults)